lib/util/toastx.dart

```diff
@@ -17,4 +17,23 @@ extension ToastX on BuildContext {
           style: TextStyle(color: Colors.amber[400]),
         ),
       ).show(this);
+
+  void showCommonWarningToast({
+    required String title,
+    required String desc,
```

Comment:
please use full words when naming anything, e.g.: `description` instead of `desc`